packages/graphql-transformers-e2e-tests/src/__tests__/KeyTransformerLocal.e2e.test.ts

```diff
@@ -346,7 +346,7 @@ test('Test that connection type is generated for custom query when queries is se
     `;
 
   const transformer = new GraphQLTransform({
-    transformers: [new ModelTransformer(), new KeyTransformer()],
+    transformers: [new DynamoDBModelTransformer(), new KeyTransformer()],
```

Comment:
Did we change the name of the Transformer? There are many folks out here who use the transformer lib directly..so would this be a breaking change for them?